src/components/toast/toast.js

```diff
@@ -213,7 +213,9 @@ function MdToastProvider($$interimElementProvider) {
         var opts = {
           template: [
             '<md-toast md-theme="{{ toast.theme }}" ng-class="{\'md-capsule\': toast.capsule}">',
-              '<span flex>{{ toast.content }}</span>',
+              '<span flex role="alert" aria-relevant="all" aria-atomic="true">' +
```

Comment:
That's good, there is some support then! I did find the toasts were not announced in JAWS with IE or Chrome and Voiceover. JAWS would be more important, while Chrome isn't used with VO as much as Safari.